edalize/vcs.py

```diff
@@ -0,0 +1,86 @@
+import os
+import logging
+
+from edalize.edatool import Edatool
+
+logger = logging.getLogger(__name__)
+
+class Vcs(Edatool):
+
+    tool_options = {
+        'lists' : {'vcs_options' : 'String'}
+    }
+
+    argtypes = ['plusarg', 'vlogdefine', 'vlogparam']
+
+    MAKEFILE_TEMPLATE = """
+all: $(TARGET)
+
+$(TARGET):
+	vcs -full64 -t $(TOPLEVEL) -f $(TARGET).scr -o $@ $(VCS_OPTIONS)
+
+run: $(TARGET)
+	$(TARGET) -l vcs.log $(EXTRA_OPTIONS)
+
+clean:
+	$(RM) $(TARGET)
+"""
```

Comment:
I would love if no more inline Makefiles are introduced and templates would be used instead. The Spyglass backend is one of the most recent ones and could be a good starting point.

Reply:
Switched to using a template instead! Would be good if you could give it a look over as I've never used jinja2.

Reply:
My patience with jinja2 is growing thin, and also, it's likely that the makefiles will be replaced by platform-specific .bat/.sh files in the near future, so I don't think we should spend too much time on this. If the template works, let's keep it. Otherwise revert to the inline makefile